vm_scripts/add_disk.py

```
@@ -0,0 +1,272 @@
+from pyVmomi import vim, vmodl
```

Ruff (0.8.2)

`pyVmomi.vmodl` imported but unused

Remove unused import: `pyVmomi.vmodl`

(F401)